doc/api/stream.md

```diff
@@ -1445,9 +1445,9 @@ added: v8.0.0
 -->
 
 * `callback` {Function} Call this function (optionally with an error
-  argument) when you are done writing any remaining data.
+  argument) when finished writing any remaining data.
 
-Note: `_final()` **must not** be called directly.  It MAY be implemented
+*Note*: `_final()` **must not** be called directly. It MAY be implemented
```

Comment:
Second optional nit: Is this really a "note"? Seems like it could just be a sentence by itself without prepending `Note:`.